src/testUtils/fixtures/setupTrust.js

```diff
@@ -0,0 +1,14 @@
+import AppContainer from "../../containers/AppContainer";
+const container = AppContainer.getInstance();
+
+export default async ({
+  name = null,
+  adminCode = null,
+  password = null,
+} = {}) => {
+  return await container.getCreateTrust()({
+    name: name || "Test Trust",
+    adminCode: adminCode || "TESTCODE",
+    password: password || "TESTPASSWORD",
+  });
```

Comment:
Nice. You can also apply a similar pattern here to prevent repeating keys
```javascript
export default sync (args = {}) => {
    return await container.getCreateTrust()({
        name: 'Test Trust',
        adminCode: 'TESTCODE',
        password: 'TESTPASSWORD',
        ...args
    });
};
```

Though not sure if you give up some intellisense hints doing that